Submitty/Submitty
# [Testing:Calendar] Better Calendar Sample Data

### Please check if the PR fulfills these requirements:


### What is the current behavior?
No dates on Calendar except for Tutorial due dates which are all on the same day

### What is the new behavior?
Fixes #9514 
![image](https://github.com/Submitty/Submitty/assets/47994802/9f296247-ad35-43f3-8b57-65ce9108e62f)
Changes gradeable dates in Development and Tutorial in order to populate the Calendar with data in order to help development

### How to test:

1. Run recreate sample courses
2. Check if the Calendar is populated with dates
3. Ensure development and tutorial course is ordered alphabetically
